src/controllers/v1/mentors.js

```diff
@@ -76,7 +77,7 @@ module.exports = class Mentors {
 
 	async share(req) {
 		try {
-			const shareLink = await mentorsHelper.share(req.params.id)
+			const shareLink = await userProfile.share(req.params.id)
```

Comment:
@nevil-mathew you should acces mentorsHelper only.

Reply:
hey..you asked me to move that function into userProfile in the previous PR.

Reply:
Hey made the changes.